sentry/src/main/java/io/sentry/clientreport/DiscardReason.java

```diff
@@ -0,0 +1,22 @@
+package io.sentry.clientreport;
+
+public enum DiscardReason {
+  QUEUE_OVERFLOW("queue_overflow"),
+  CACHE_OVERFLOW("cache_overflow"),
+  RATELIMIT_BACKOFF("ratelimit_backoff"),
+  NETWORK_ERROR("network_error"),
+  SAMPLE_RATE("sample_rate"),
+  BEFORE_SEND("before_send"),
+  EVENT_PROCESSOR("event_processor"), // also for ignored exceptions
```

Comment:
Nit: Formatting `EVENT_PROCESSOR("event_processor");`

Reply:
hmm good catch, don't know how that happened